### PR TITLE
Using store-icons instead of dreamstore-icons

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?
As the title says.

#### How should this be manually tested?
https://iconsaga--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/LogoutCustomerSession.tsx [hunk 1]
```diff
@@ -100,15 +109,17 @@ export default class LogoutCustomerSession extends Component<Props> {
           {mobile ? this.clientName : client.email}
         </div>
       </div>
-    )
-  }
+    );
```
Comment (kevinch) [inline react/components/LogoutCustomerSession.tsx]: Are you sure about the addition of semi colons back?

Comment (matheusps) [reply react/components/LogoutCustomerSession.tsx]: Oh, the `.prettierrc` is missing in this component. i'll add that

File: react/typings/vtex.dreamstore-icons.d.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-declare module 'vtex.dreamstore-icons' {
-  import { ReactElement } from 'react'
+declare module "vtex.store-icons" {
+  import { ReactElement } from "react";
```
Comment (rerissondaniel) [inline react/typings/vtex.dreamstore-icons.d.ts]: Should run prettier on this too

File: react/index.tsx [hunk 3]
```diff
@@ -12,17 +12,17 @@ import processSession from './utils/processSession'
 
 interface Props {
   /** Intl object */
-  intl: any,
+  intl: any
```
Comment (kevinch) [inline react/index.tsx]: Shouldn't `intl` be of type `intlShape`?

File: react/components/LoginAsCustomer.tsx [hunk 4]
```diff
@@ -35,12 +35,23 @@ export default class LoginAsCustomer extends Component<Props> {
 
     return (
       <div className={`${telemarketing.login}`}>
```
Comment (kevinch) [inline react/components/LoginAsCustomer.tsx]: Pretty sure `className={telemarketing.login}` is enough here.

File: react/components/LoginAsCustomer.tsx [hunk 5]
```diff
@@ -35,12 +35,23 @@ export default class LoginAsCustomer extends Component<Props> {
...
+          arrowClasses="bg-base--inverted"
+          renderHeader={this.handleHeaderRendering}
+        >
           <div className="bg-base--inverted w-100 pa4">
             <div className={`${telemarketing.popoverHeaderIcon}`}>
-              <IconAssistantSales size={50} viewBox="0 0 21 21" activeClassName="white" />
+              <IconAssistantSales
+                size={50}
+                viewBox="0 0 21 21"
+                activeClassName="white"
```
Comment (kevinch) [inline react/components/LoginAsCustomer.tsx]: Shouldn't this be a token?

Comment (brunojdo) [reply react/components/LoginAsCustomer.tsx]: yes

File: react/typings/vtex.dreamstore-icons.d.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
-declare module 'vtex.dreamstore-icons' {
+declare module 'vtex.store-icons' {
```
Comment (brunojdo) [inline react/typings/vtex.dreamstore-icons.d.ts]: Change the filename

File: react/components/LoginAsCustomer.tsx (hunk 5 above)
Comment (matheusps) [reply react/components/LoginAsCustomer.tsx]: Done @ #35

File: react/typings/vtex.dreamstore-icons.d.ts (hunk 2 above)
Comment (matheusps) [reply react/typings/vtex.dreamstore-icons.d.ts]: done

File: react/components/LoginAsCustomer.tsx (hunk 4 above)
Comment (matheusps) [reply react/components/LoginAsCustomer.tsx]: will change on #35

**Pull request merged**